specs/endpoints/food_create_spec.js

```diff
@@ -0,0 +1,55 @@
+var shell = require('shelljs');
+var request = require("supertest");
+var app = require('../../app');
+
+// describe('Test POST /api/v1/foods path', () => {
+//   test('it should return the successfully created food', () => {
+//     return request(app).post("/api/v1/foods").then(response => {
+//       expect(response.statusCode).toBe(200)
+//     })
+//   })
+// })
+describe('Food create API', () => {
+  beforeAll(() => {
+    shell.exec('npx sequelize db:create')
+  });
+  beforeEach(() => {
+      shell.exec('npx sequelize db:migrate')
+      shell.exec('npx sequelize db:seed:all')
+    });
+  afterEach(() => {
+    shell.exec('npx sequelize db:seed:undo:all')
+  });
+
+  // describe('Test POST /api/v1/foods path' , () => {
+  //   test('it should create a food', async() => {
+  //     const newFood =  {
+  //       name: "Pringles",
+  //       calories: 27
+  //      }
+  //      try {
+  //        const response = await request(app).post('/api/v1/service/foods').send(newFood)
+  //        expect(response.statusCode).toBe(20)
+  //      } catch (err) {
+  //        // write some test here
+  //        expect(response.statusCode).toBe(500)
+  //      }
+  //     }
+  //   })
+  // })
+  describe('Test POST /api/v1/foods path' , () => {
+    test('it should create a food successfully', () => {
+      const newFood =  {
+        name: "Pringles",
+        calories: 27
+      }
+      return request(app).post("/api/v1/foods").then(response => {
+        expect(response.status).toBe(20)
+      })
+    })
+
```

Comment:
It looks almost all right. Believe you need to send the newFood int he body. So try this for line 46:
```return request(app).post("/api/v1/foods").send(newFood).then(response => { ```

Reply:
Alternatively/additionally, can try this syntax for async/await:
```test('it should return a food object', async () => {
      await request(app).get("/api/v1/foods/1").then(async response => {
        await expect(response.body.id).toBe(1),
        await expect(response.body.name).toBe('Cheetos'),
        await expect(response.body.calories).toBe(30)
      });
    });
```

Reply:
I'm not seeing what's different between the code you have for line 48 and the code I have. Could be a version thing... ?

Reply:
the "send(NewFood)". You need to send your request body (what you created) somehow.